### DISCUSSION TITLE
How can I use "Transitions"?

### DISCUSSION THREAD

**Discussion by hidech** [webstudio-is/webstudio-community#135] [Q&A] [answered]
I want to use transitions to animate some elements.
How can I use Transitions option?
I don't see difference when I set the option value.
Thanks in advance.

**Comment by kof** [comment, marked as answer]
Transitions are set on a regular state, then you define the styles on hover or other states so that transition can transition from one state to another, this is not specific to webstudio, it's how CSS works.

**Reply by hidech** [reply]
Thanks for your response. I know.

**Reply by kof** [reply]
you mean you did that and it didn't work? in that case please share specific project with minimal reproduction

**Reply by hidech** [reply]
I want to animate specific element when a page loaded, but have no idea, something like this:

```
    .fade-in-float {
        opacity: 0;
        transform: translateY(5px);
        transition: opacity 0.5s ease, transform 0.5s ease;
    }
    .fade-in-float.fade-in {
        opacity: 1;
        transform: translateY(0);
    }
```

**Reply by kof** [reply]
page load is an event CSS doesn't have and neither does builder, this is only possible if somebody adds a style or class to the element. I think you are better off doing it with that code in html embed right now

**Reply by kof** [reply]
generally this will be possible in the future, with added event system, but not yet

**Reply by hidech** [reply]
Yes, of course. Thanks for your information to be clear.
